TEveEventDisplay/src/TEveMu2eMCInterface.cc

```diff
@@ -220,11 +198,13 @@ namespace mu2e{
               
               string energy = to_string(points[0].kineticEnergy());
    
-              const std::string title = " MCTrajectory "+ energy;// + "PDG Code = " + pdgId;
+              const std::string title = " MCTrajectory "+ energy + " Creation code = " + to_string(trajectoryIter->first->creationCode()) + "Stopping code = " + to_string(trajectoryIter->first->stoppingCode()) + " End Global Time = " + to_string(trajectoryIter->first->endGlobalTime())  ;
               line->SetTitle(Form(title.c_str()));
               
               //Get PID label:
               TEveText *t = GetLabel(trajectoryIter->first->pdgId(), line, line_twoDXZ, line_twoDXY);
+              line_twoDXZ->SetTitle(Form(title.c_str()));
+              line_twoDXY->SetTitle(Form(title.c_str()));
               line->SetLineWidth(3);
```

Comment:
Make the line width a constexpr local variable.   If you decide to change the line width you now need to edit it in many places, not 1.

Reply:
Can do